R/model.R

```diff
@@ -37,10 +41,12 @@ enw_priors <- function() {
       "Zero truncated normal",
       "Zero truncated normal",
       "Zero truncated normal",
+      "Zero truncated normal",
+      "Normal",
       "Zero truncated normal"
     ),
-    mean = c(0, 1, 0.5, rep(0, 4)),
-    sd = rep(1, 7)
+    mean = c(0, 1, 0.5, rep(0, 4), 0, 0),
+    sd = c(rep(1, 7), 1, 0.1)
```

Comment:
I was not 100% sure what sensible default priors to put here. Using an sd of 1 for the random walk seemed way to wide for me (would include up to a 7-fold increase in odds ratio in one time step in its 96% interval)... The current starting value prior is roughly between 10% and 90%.

Reply:
Yeah. I think this is fine for now. In general, going to need to think more about priors as it would be good to be able to pass effect size priors. I've done some work on this in my in progress `epict` package that I think can be ported. Not for this release though. Before resolving this someone (me!) should open an issue to do this.